src/app/sales-search/sales-search.component.ts

```diff
@@ -0,0 +1,65 @@
+import { Component, OnInit } from '@angular/core';
+import { SPACE } from '@angular/cdk/keycodes';
+import { MatChipInputEvent } from '@angular/material';
+
+@Component({
+  selector: 'app-sales-search',
+  templateUrl: './sales-search.component.html',
+  styleUrls: ['./sales-search.component.scss']
+})
+export class SalesSearchComponent implements OnInit {
```

Comment:
The component name should rather describe what it can do instead of who will be using it. To be consistent with the route naming it could be `UserProfileSearchComponent`.

Reply:
Thank you. Renamed